versions/3.1.0.md

```diff
@@ -2687,23 +2691,23 @@ Field Name | Type | Description
 
 This object MAY be extended with [Specification Extensions](#specificationExtensions).
 
-The discriminator object is legal only when using one of the composite keywords `oneOf`, `anyOf`, `allOf`.
+##### Discriminator Object Example
```

Comment:
Added this section title to clearly differentiate between the normative statements versus the example section.